### PR TITLE
oopsy: add p1n

### PR DESCRIPTION
**Pull request by quisquous**

### CODE REVIEW

File: ui/oopsyraidsy/data/06-ew/raid/p1n.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  soloWarn: {
+    'P1N True Holy': '65E7', // Pitiless Flail stack marker
+  },
+  triggers: [
+    {
+      id: 'P1N Cold Spell Track',
+      type: 'GainsEffect',
+      // No need to track this falling off or deleting it.
+      // When the next round comes along, it will reassign and give it to somebody.
+      // TODO: not sure if somebody dies if this falls off and so maybe we should track losing it?
```
Comment (JLGarber) [inline ui/oopsyraidsy/data/06-ew/raid/p1n.ts:30]: I *think* you're correct, we don't need to track losing the debuffs. Unfortunately I don't have anything in my logs for "people being dead during hot/cold".

**Review by JLGarber — COMMENTED** [review]
Is it worth adding a `shareWarn` for the Pitiless Plail knockback as well?

File: ui/oopsyraidsy/data/06-ew/raid/p1n.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+import ZoneId from '../../../../../resources/zone_id';
+import { OopsyData } from '../../../../../types/data';
+import { OopsyTriggerSet } from '../../../../../types/oopsy';
+
+export interface Data extends OopsyData {
+  spell?: { [name: string]: 'cold' | 'hot' };
+}
+
+const triggerSet: OopsyTriggerSet<Data> = {
+  zoneId: ZoneId.AsphodelosTheFourthCircle,
```
[the remark this answers is not in the document]
Comment (quisquous) [reply ui/oopsyraidsy/data/06-ew/raid/p1n.ts:11]: > Is it worth adding a `shareWarn` for the Pitiless Plail knockback as well?

I don't think so, as somebody was saying elsewhere that if enough people take it, it's not a knockback?

[the remark this answers is not in the document]
Comment (JLGarber) [reply ui/oopsyraidsy/data/06-ew/raid/p1n.ts]: I didn’t know that! I’ll have my group try it tonight.

File: ui/oopsyraidsy/data/06-ew/raid/p1n.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      id: 'P1N Pitiless Flail Knockback',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '65E5' }),
+      deathReason: (_data, matches) => {
+        return {
+          id: matches.targetId,
+          name: matches.target,
+          text: {
+            en: 'Pushed into wall',
+            de: 'Rückstoß in die Wand',
```
Comment (VanSlanzar) [inline ui/oopsyraidsy/data/06-ew/raid/p1n.ts:78]: `fr: 'Poussé(e) dans le mur',`

if you want add this translation. :)

Comment (quisquous) [reply ui/oopsyraidsy/data/06-ew/raid/p1n.ts:78]: Please submit another pull request, as this probably exists elsewhere too.

**Pull request merged**